docs/semgrep-supply-chain/getting-started.md

```diff
@@ -52,17 +52,17 @@ If you have code files in `my-project/biking`, Semgrep Supply Chain does not ass
 
 ## Scan frequency
 
-By default, Semgrep Supply Chain scans your code once per day. However, you can change this so Semgrep Supply Chain scans your code at a different frequency or when a specific event occurs.
+By adjusting your CI configuration, you can configure your scans so that Semgrep Supply Chain scans your code at a different frequency or when a specific event occurs.
 
 ### Schedule scans
 
-Semgrep Supply Chain frequently receives rule updates. To take advantage of these updates and increase the frequency with which Semgrep Supply Chain scans your codebase.
+Semgrep Supply Chain frequently receives rule updates. To take advantage of these updates, adjust the frequency with which Semgrep Supply Chain scans your codebase.
 
 <CiScheduling />
 
 :::note Rules updates
 
-If a rule is updated, findings generated against the updated rule are considered **new findings**, even if the previous version of the rule generated a finding. Furthermore, because the finding is new, you'll receive notifications through the channels you've set up (e.g., Slack, email).
+If a rule is updated, findings generated against the updated rule are considered **new findings**, even if the previous version of the rule generated a finding. Because the finding is new, you'll receive notifications through the channels you've set up (e.g., Slack, email).
```

Comment:
@andy-r2c This line had stuck with me since I made a wording change unrelated to the content, but based on separate discussion it sounds like it might not even be correct? Could you confirm?